### PR TITLE
Enables dw.js configruation and updates docs

### PR DESCRIPTION
**Pull request by Eneris**
Adds support for `dw.js` config

### CODE REVIEW

**Comment by Eneris** [comment]
Solves #207

File: src/server/WebDav.ts [hunk 1]
```diff
@@ -388,34 +388,39 @@ export default class WebDav {
...
 	return new Observable<DavOptions>(observer => {
-		const stream = createReadStream(configFilename);
-		let chunks: Buffer[] = [];
-
-		// Listen for data
-		stream.on('data', chunk => {
-			chunks.push(chunk);
-		});
+		if (configFilename.match(/\.js$/)) {
+			observer.next(require(configFilename));
```
Comment (SqrTT) [inline src/server/WebDav.ts]: Hi @Eneris 
Thank you for submitting PR. I'll merge PR but need small adjustment here. I as user definitely make some typo or/and other mistake in `dw.js`, So would be great to handle this case, let's say via, `try ... catch`.

Also there are small disadvantage of using `require`: it caches module until app is restarted, so if you change credentials VS Code may not see it until restart. I'm fine with this and we may figureout something later.

Comment (Eneris) [reply src/server/WebDav.ts]: * Added `try ... catch`
* Invalidating module cache in every request

**Comment by SqrTT** [comment]
Excellent! Thank you very much!

**Pull request merged**